### DISCUSSION TITLE
Document Ready Not Firing on page change.

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#140] [1. Help] [answered]
Hi, 
I have observed that If we write any code in document.ready OR window.load and then move through pages like from Homepage to Category page. That code doesn't fire. When we do so a blue color progress bar shows on top which looks like the page is loaded through ajax or something.

That code only fires when we are on a page and reload that page. In this case the progress bar doesn't show and in favicon icon place we can see that page is re-loaded.

Doesn't breeze reload a page when we go to the page through navigation or link?

**Comment by dheerajKsharma** [comment, marked as answer]
Ok got it. It was turbo mode feature of breeze. I disabled it now. Working fine.

**Reply by vovayatsyuk** [reply]
You can also listen to the `breeze:load` event. It's compatible with turbo and no-turbo modes:

```js
$(document).on('breeze:load', function () {
    console.log('ready!');
});
```

Source: https://breezefront.com/docs/events#breezeload
